wd_core.au3

```diff
@@ -1738,18 +1736,12 @@ EndFunc   ;==>__WD_CloseDriver
 ;                  $iOption - [optional] Any combination of $JSON_* constants. Default is 0.
 ; Author ........: Danp2
 ; Modified ......:
-; Remarks .......: $JSON_MLREFORMAT will strip tabs and CR/LFs from a multiline string.
-;                  See $JSON_* constants in json.au3 for other $iOption possibilities.
+; Remarks .......:
```

Comment:
this remark
`See $JSON_* constants in json.au3 for other $iOption possibilities.`

should stay here.